ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ClusterPassivationStoreResourceDefinition.java

```diff
@@ -57,7 +57,8 @@
     static final SimpleAttributeDefinition CACHE_CONTAINER = new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.CACHE_CONTAINER, ModelType.STRING, true)
             .setXmlName(EJB3SubsystemXMLAttribute.CACHE_CONTAINER.getLocalName())
             .setDefaultValue(new ModelNode(BeanManagerFactoryServiceConfiguratorConfiguration.DEFAULT_CONTAINER_NAME))
-            .setAllowExpression(true)
+            // Capability references should support expressions
+            .setAllowExpression(false)
```

Comment:
the doc "Capability references should support expressions" seems to contradict with the new change. Did you mean "should not"?

Reply:
@chengfang Thanks, yes, that's clearly a typo. Fixed now!